pkg/server/commands/write.go

```diff
@@ -79,26 +75,6 @@ func (c *WriteCommand) validateWriteRequest(ctx context.Context, req *openfgav1.
 			if err != nil {
 				return serverErrors.ValidationError(err)
 			}
-
-			objectType, _ := tupleUtils.SplitObject(tk.GetObject())
-
-			relation, err := typesys.GetRelation(objectType, tk.GetRelation())
-			if err != nil {
-				if errors.Is(err, typesystem.ErrObjectTypeUndefined) {
-					return serverErrors.TypeNotFound(objectType)
-				}
-
-				if errors.Is(err, typesystem.ErrRelationUndefined) {
-					return serverErrors.RelationNotFound(tk.GetRelation(), objectType, tk)
-				}
-
-				return serverErrors.HandleError("", err)
-			}
```

Comment:
This is already here https://github.com/openfga/openfga/blob/8dc2ed62413e7cccc7da57da057f1f1fc17dcb8a/internal/validation/validation.go#L206-L230